VideoLocker/res/layout/fragment_course_combined_info.xml

```diff
@@ -21,7 +21,7 @@
         <RelativeLayout
             android:id="@+id/announcement_layout"
             android:layout_width="match_parent"
-            android:layout_height="wrap_content">
+            android:layout_height="fill_parent">
```

Comment:
Presumably, we'd add it to a view higher in the hierarchy or something like that?